content/en/platform/corda/4.4/enterprise/release-notes-enterprise.md

```diff
@@ -19,6 +19,11 @@ weight: 10
 
 # Corda Enterprise Edition 4.4 release notes
 
+Corda 4.4 lays the foundation of a new open-core approach for the Corda codebase. This involved a refactoring of the main functional components of Corda. Please consult cordapp-overview.rst to get an overview of the practical impact on CorDapp development.
```

Comment:
' open-core ' ?

Reply:
I presume it refers to this:

https://en.wikipedia.org/wiki/Open-core_model

Shall I say "new open-core model approach"?

Reply:
I think it's better ;)